app/models/review.rb

```diff
@@ -10,11 +10,16 @@ class Review < ActiveRecord::Base
 
   belongs_to :venue
   belongs_to :user
+  has_many :votes
 
+  include PgSearch
+  pg_search_scope :search_by_review_body,
+    against: :review_body,
+    :using => {
+                :tsearch => {:prefix => true}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Space inside { missing.<br>Space inside } missing.